### PR TITLE
Feat/start cmd

### PR DESCRIPTION
**Pull request by fengtality**
**Before submitting this PR, please make sure**:

- [X] Your code builds clean without any errors or warnings
- [X] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**A description of the changes proposed in the pull request**:

- Adds `start` command for users running from source
- Changes Dockerfile

**Tips for QA testing**:

- Pull `start-cmd` Docker image which uses this build

### CODE REVIEW

**Comment by rapcmia** [comment]
PR update:
- It fixes the issue https://github.com/hummingbot/hummingbot/issues/6513
- Client now can be started using `./start` command ✅ 
- Build docker image and observe if there are any issue, all ok 

---

However if a user or a development update caused issue that could not let the client start, the error and traceback we need is hidden due to the `2>/dev/null`. 
- For source build: We can use command `bin/hummingbot.py` to start the client so we see the error
- For docker build: 

   https://github.com/hummingbot/hummingbot/assets/73840223/f4c4de88-be7c-43c9-870c-ebc8423756c7

   - If the container is not running after successfully created on bash_scripts or docker compose, the common advise we do is to run `docker logs container_name` to look for possible error. However on this PR and recording above due to the added `2>/dev/null` it is now hidden.

Steps to reproduce:
1. For source build, try to add  extra characters on any file codebase
2. For docker build, after. successfully created and opened the container:
    - Close it again and delete `conf` folder
    - Start the docker container again and observe behavior (should not be displayed on `docker ps`)
    - Run `docker logs container_name`


Note: This is PR is good to be approve however would need some inputs regarding the possible behavior above, thank you 🙇🏼

**Comment by fengtality** [comment]
@rapcmia I added a commit that should append errors to `errors.log` instead of overriding them. Does that fix the Docker issue?

**Comment by nikspz** [comment]
Note: Crosschecked both development and PR6518 on source, both don’t have **Error parsing 'content-type'** after hour after Client started

PR6518 source:


- Test performed
    - Source build
        - Run .`/start`, terminal logs returns:
        
        ```jsx
        AsyncClient - chain session cookie loaded from disk:
        trading_pair_fetcher - An error occurred when fetching trading pairs for injective_v2.Please check the logs (See log file for stack trace dump)
        ```
        
        - Created error.log file which is `empty` ❌ However messages recorded at  `logs_hummingbot.log`
        ```jsx
        2023-08-21 09:57:20,087 - 657898 - pyinjective.async_client.AsyncClient - INFO - chain session cookie loaded from disk: 
        2023-08-21 09:57:20,098 - 657898 - hummingbot.core.utils.trading_pair_fetcher - ERROR - An error occurred when fetching trading pairs for injective_v2.Please check the logs
        Traceback (most recent call last):
          File "/home/nikita/start6518/hummingbot/core/utils/trading_pair_fetcher.py", line 53, in fetch_all
            self._fetch_pairs_from_connector_setting(connector_setting=conn_setting)
          File "/home/nikita/start6518/hummingbot/core/utils/trading_pair_fetcher.py", line 38, in _fetch_pairs_from_connector_setting
            connector = connector_setting.non_trading_connector_instance_with_default_configuration()
          File "/home/nikita/start6518/hummingbot/client/settings.py", line 334, in non_trading_connector_instance_with_default_configuration
            connector = connector_class(**kwargs)
          File "/home/nikita/start6518/hummingbot/connector/exchange/injective_v2/injective_v2_exchange.py", line 60, in __init__
            self._data_source = connector_configuration.create_data_source()
          File "/home/nikita/start6518/hummingbot/connector/exchange/injective_v2/injective_v2_utils.py", line 319, in create_data_source
            return self.account_type.create_data_source(
          File "/home/nikita/start6518/hummingbot/connector/exchange/injective_v2/injective_v2_utils.py", line 205, in create_data_source
            return InjectiveGranteeDataSource(
          File "/home/nikita/start6518/hummingbot/connector/exchange/injective_v2/data_sources/injective_grantee_data_source.py", line 58, in __init__
            self._private_key = PrivateKey.from_hex(private_key)
          File "/home/nikita/miniconda3/envs/hummingbot/lib/python3.10/site-packages/pyinjective/wallet.py", line 75, in from_hex
            self.signing_key = SigningKey.from_string(bytes.fromhex(priv), curve=SECP256k1, hashfunc=hashlib.sha256)
          File "/home/nikita/miniconda3/envs/hummingbot/lib/python3.10/site-packages/ecdsa/keys.py", line 916, in from_string
            return cls.from_secret_exponent(secexp, curve, hashfunc)
          File "/home/nikita/miniconda3/envs/hummingbot/lib/python3.10/site-packages/ecdsa/keys.py", line 857, in from_secret_exponent
            raise MalformedPointError(
        ecdsa.errors.MalformedPointError: Invalid value for secexp, expected integer between 1 and 115792089237316195423570985008687907852837564279074904382605163141518161494337
        ```
        
        - Run `bin/humminbot.py`: same error/warning message `error occurred when fetching trading pairs for injective_v2` on terminal
            - error.log file not created, however this messages recorded at `logs_hummingbot.log`
        
        ```jsx
        2023-08-21 09:47:17,927 - 657805 - pyinjective.async_client.AsyncClient - INFO - chain session cookie loaded from disk: 
        2023-08-21 09:47:17,931 - 657805 - hummingbot.core.utils.trading_pair_fetcher - ERROR - An error occurred when fetching trading pairs for injective_v2.Please check the logs
        Traceback (most recent call last):
          File "/home/nikita/start6518/hummingbot/core/utils/trading_pair_fetcher.py", line 53, in fetch_all
            self._fetch_pairs_from_connector_setting(connector_setting=conn_setting)
          File "/home/nikita/start6518/hummingbot/core/utils/trading_pair_fetcher.py", line 38, in _fetch_pairs_from_connector_setting
            connector = connector_setting.non_trading_connector_instance_with_default_configuration()
          File "/home/nikita/start6518/hummingbot/client/settings.py", line 334, in non_trading_connector_instance_with_default_configuration
            connector = connector_class(**kwargs)
          File "/home/nikita/start6518/hummingbot/connector/exchange/injective_v2/injective_v2_exchange.py", line 60, in __init__
            self._data_source = connector_configuration.create_data_source()
          File "/home/nikita/start6518/hummingbot/connector/exchange/injective_v2/injective_v2_utils.py", line 319, in create_data_source
            return self.account_type.create_data_source(
          File "/home/nikita/start6518/hummingbot/connector/exchange/injective_v2/injective_v2_utils.py", line 205, in create_data_source
            return InjectiveGranteeDataSource(
          File "/home/nikita/start6518/hummingbot/connector/exchange/injective_v2/data_sources/injective_grantee_data_source.py", line 58, in __init__
            self._private_key = PrivateKey.from_hex(private_key)
          File "/home/nikita/miniconda3/envs/hummingbot/lib/python3.10/site-packages/pyinjective/wallet.py", line 75, in from_hex
            self.signing_key = SigningKey.from_string(bytes.fromhex(priv), curve=SECP256k1, hashfunc=hashlib.sha256)
          File "/home/nikita/miniconda3/envs/hummingbot/lib/python3.10/site-packages/ecdsa/keys.py", line 916, in from_string
            return cls.from_secret_exponent(secexp, curve, hashfunc)
          File "/home/nikita/miniconda3/envs/hummingbot/lib/python3.10/site-packages/ecdsa/keys.py", line 857, in from_secret_exponent
            raise MalformedPointError(
        ecdsa.errors.MalformedPointError: Invalid value for secexp, expected integer between 1 and 115792089237316195423570985008687907852837564279074904382605163141518161494337
        ```
        
- Docker build:
    - Manually created docker image successfully
    - Run docker hummingbot-create script, container created successfully
    - Started instance, review logs folder:
        - empty `error.log` file created ❌
        - logs recorded at `logs_hummingbot.log`
        - Deleted conf folder and restart docker start sta && docker attach sta
            - review instance started successfully and conf folder recreated

**Review by nikspz — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -73,4 +73,4 @@ SHELL [ "/bin/bash", "-lc" ]
 
 # Set the default command to run when starting the container
 
-CMD conda activate hummingbot && ./bin/hummingbot_quickstart.py 2>./logs/standard_error_output.txt
\ No newline at end of file
+CMD conda activate hummingbot && ./bin/hummingbot_quickstart.py 2>> ./logs/errors.log
```
Comment (cardosofede) [inline Dockerfile:76]: @fengtality I reverted the changes because in the last commit you changed ./bin/hummingbot.py for ./bin/hummingbot_quickstart.py

**Review by cardosofede — APPROVED** [review]
LGTM!

**Comment by cardosofede** [comment]
@fengtality @nikspz @rapcmia I did some changes because the Dockerfile was using hummingbot.py instead of hummingbot_quickstart.py and that won't allow the autostart of strategies.
Also I removed the touch because if the file does not exist the bash command will create it.
QA testing before merging tomorrow:
- Run it from source
- Build docker image and run it in a clean folder

After that we are ready to merge

**Comment by rapcmia** [comment]
Merged to development and part of release version 1.19.0

**Pull request merged**